csharp/ql/test/library-tests/statements/Using1.ql

```diff
@@ -1,8 +1,12 @@
 /**
  * @name Test for usings
  */
+
 import csharp
 
-where forall(UsingStmt s | exists(s.getAnExpr())
-          and exists(s.getBody()))
+where
+  forall(UsingStmt s |
+    exists(s.getAnExpr()) and
+    exists(s.getBody())
+  )
```

Comment:
@aschackmull : Shouldn't this be formatted on a single line?

Reply:
The autoformatter actually allows both.  For conjunctions and disjunctions the autoformatter allows single-line formatting if the terms are sufficiently short and compact.  But the QL author always has the option of splitting conjunctions and disjunctions across multiple lines if so desired, and that's what happens here: The original conjunction is split across two lines.  If you were to join just the conjunction part on a single line and then rerun the autoformat, then I'd guess that it would put the entire `forall(..)` on one line.